document-generator-accounts/src/test/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImplTest.java

```diff
@@ -27,40 +28,47 @@
     @InjectMocks
     private AccountsDocumentInfoServiceImpl accountsDocumentInfoService;
 
+    @Mock
+    private AccountsHandler accountsHandlerMock;
+
     @Mock
     private TransactionService transactionService;
 
+    @Test
+    @DisplayName("Tests the unsuccessful retrieval of an document data due to an error in transaction retrieval")
+    void testUnsuccessfulGetDocumentInfoFailedTransactionRetrieval() {
+        when(transactionService.getTransaction(anyString())).thenReturn(null);
+        assertNull(accountsDocumentInfoService.getDocumentInfo());
+    }
+
     @Test
     @DisplayName("Tests the unsuccessful retrieval of an accounts document data due to no accounts resource in transaction")
     void testUnsuccessfulGetDocumentInfoNoAccountsResourceInTransaction() {
-        Map<String, Resource> resources = new HashMap<>();
-        resources.put("error", createResource());
-
-        Transaction transaction = new Transaction();
-        transaction.setResources(resources);
+        Transaction transaction = createTransaction();
+        transaction.getResources().remove("");
```

Comment:
Can you use a test ID instead of an empty string please? It'll be easier to see the use of the resource ID in the tests if it's a constant instead of duplicating the empty string.